src/libraries/System.Reflection.MetadataLoadContext/tests/ILLink.Descriptors.xml

```diff
@@ -0,0 +1,3 @@
+<linker>
+  <assembly fullname="mscorlib" />
```

Comment:
Is this still necessary with the latest linker? It should be preserving facades like this.

Reply:
This was needed with whatever linker is being used with `main` from a couple of days ago.